### PR TITLE
feat: Add $attrs to input field

### PR DESCRIPTION
**Pull request by khamer**
This passes through `$attrs` to the `<input>` element per Issue #25. I removed a bunch of the props that aren't necessary anymore because of it too, as well as added a couple examples to the index.html.

It passes linting, but not unit tests - but I couldn't get the unit tests to run before this PR either.

Let me know if there's anything you'd like me to change.

### CODE REVIEW

**Comment by apertureless** [comment]
Very cool! Will look the coming days into it!

**Comment by apertureless** [comment]
Nice, I will check it out the coming days!

File: src/components/PasswordStrengthMeter.vue [hunk 1]
```diff
@@ -5,24 +5,19 @@
       class="Password__group"
     >
       <input
+        v-bind="$attrs"
         :type="inputType"
-        :ref="referanceValue"
+        :ref="referenceValue"
         :class="[defaultClass, disabled ? disabledClass : '']"
```
Comment (raybotha) [inline src/components/PasswordStrengthMeter.vue]: 👍  This is working nicely for me, besides probably needing this change:
```suggestion
        :class="[defaultClass, $attrs.disabled ? disabledClass : '']"
```

**Comment by khamer** [comment]
Yeah I agree @raybotha - I pulled in your change.

**Pull request merged**